contracts/src/Types.sol

```diff
@@ -76,19 +76,17 @@ enum OperatingMode {
 
 /// @dev Messages from Polkadot take the form of these commands.
 enum Command {
-    AgentExecute,
+    TransferToken,
```

Comment:
Just realized this change is going to break compatibility, causing messages in the old format to fail processing.

So we need to some kind of solution to allow old messages and new messages to be processed.

Reply:
could it be simpler to just stick to the old message format overall, even for this work?

Reply:
https://github.com/Snowfork/snowbridge/pull/1155/commits/fcd7539afd1d7d36b1a5b3d2bf53b49c54fe956a and https://github.com/Snowfork/polkadot-sdk/pull/128/commits/40cc470c2d6fc2c6995e2e18b03d13b773f34664

Add `AgentExecuteCommand` back for compatibility. Meanwhile, make `TransferToken` also a top-level command so we can deprecate `AgentExecuteCommand` in later upgrading.